package.json

```
@@ -3,19 +3,31 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "develop": "parcel ./index.html --port 3000",
-    "build": "parcel build ./index.html",
-    "start": "parcel build ./index.html && node ./server.js"
+    "csslint": "stylelint \"**/*.scss\"",
+    "jslint": "eslint ./src/**/*.ts",
+    "dev": "parcel ./src/index.html --port 3000",
+    "build": "parcel build ./src/index.html",
+    "start": "parcel build ./src/index.html && node ./server.js"
```

Можно переиспользовать 
```suggestion
    "start": "npm run build && node ./server.js"
```